src/defines.h

```diff
@@ -26,7 +26,7 @@
 // ================================================
 
 // Set parameters of IMU and board used
-#define IMU IMU_BNO085
+#define IMU IMU_FSP201
```

Comment:
```suggestion
#define IMU IMU_BNO085
```
we should probably not default to the FSPs if we aren't going to use these in prod

Reply:
Yes, this is my error